Kgl mlst dev

kevinlibuit opened the pull request:
This PR adds mlst typing functionality to the TheiaProk_Illumina_PE workflow 
- Outputs are also included in any taxon table exports

Other modifications made:
- Exposed Abricate min len/coverage params
- Allow inclusion of other metadata fields for taxon table exports (collection_date, originating_lab, city, county, zip)

rpetit3 commented:
I don't think this will be an issue, but `kgl-theiaprok-dev` has `task_mlst.wdl`  https://github.com/theiagen/public_health_bacterial_genomics/blob/kgl-theiaprok-dev/tasks/species_typing/task_mlst.wdl

kevinlibuit commented:
@rpetit3 and it's actually the same task. After merging kgl-theiaprok-dev, I used your task `task_mlst.wdl` file and modified it slightly to create that report file and renamed to `ts_mlst.wdl` for clarity (Torsten Seeemann's mlst)
